src/vesting.cairo

```diff
@@ -65,10 +71,11 @@ mod vesting {
             vested_timestamp: u64
```

Comment:
What exactly is `vested_timestamp`?

Reply:
Time when a vesting milestone is eligible, i.e. can be vested. In this case, it is in the past, which is why I call it vested_timestamp not vesting_timestamp